src/context.c

```diff
@@ -145,6 +145,8 @@ static gboolean launch_and_wait_variables_handler(gchar *handler_name, GHashTabl
 			g_hash_table_insert(variables, g_strdup(split[0]), g_strdup(split[1]));
 			g_strfreev(split);
 		}
+
+		g_free(outline);
```

Comment:
Please additionally fix the memory leaks that occur up in [line 143's](https://github.com/rauc/rauc/blob/c441d4d2001c1413ec570bc3dc331ff85e66c23f/src/context.c#L143) `continue` statement: before continuing the do-loop a `g_strfreev(split)` and a `g_free(outline)` are missing.

Reply:
I've added another commit to also fix this by using auto ptr.